docs/admin/notifications.md

```diff
@@ -0,0 +1,251 @@
+|    -     | `--notifications-email-tls-starttls`      | `CODER_NOTIFICATIONS_EMAIL_TLS_STARTTLS`    | `bool`   | Enable STARTTLS to upgrade insecure SMTP connections using TLS. Ignored if `CODER_NOTIFICATIONS_EMAIL_FORCE_TLS` is set.                                         | false   |
+|    -     | `--notifications-email-tls-skip-verify`   | `CODER_NOTIFICATIONS_EMAIL_TLS_SKIPVERIFY`  | `bool`   | Skip verification of the target server's certificate (**insecure**).                                                                                             | false   |
+|    -     | `--notifications-email-tls-server-name`   | `CODER_NOTIFICATIONS_EMAIL_TLS_SERVERNAME`  | `string` | Server name to verify against the target certificate.                                                                                                            |         |
+|    -     | `--notifications-email-tls-cert-file`     | `CODER_NOTIFICATIONS_EMAIL_TLS_CERTFILE`    | `string` | Certificate file to use.                                                                                                                                         |         |
+|    -     | `--notifications-email-tls-cert-key-file` | `CODER_NOTIFICATIONS_EMAIL_TLS_CERTKEYFILE` | `string` | Certificate key file to use.                                                                                                                                     |         |
+
+**NOTE:** you _MUST_ use `CODER_NOTIFICATIONS_EMAIL_FORCE_TLS` if your smarthost supports TLS on a port other than `465`.
+
+### Send emails using G-Suite
+
+After setting the required fields above:
+
+1. Create an [App Password](https://myaccount.google.com/apppasswords) using the account you wish to send from
+2. Set the following configuration options:
+    ```
+    CODER_NOTIFICATIONS_EMAIL_SMARTHOST=smtp.gmail.com:465
+    CODER_NOTIFICATIONS_EMAIL_AUTH_USERNAME=<user>@<domain>
+    CODER_NOTIFICATIONS_EMAIL_AUTH_PASSWORD="<app password created above>"
+    ```
+
+See [this help article from Google](https://support.google.com/a/answer/176600?hl=en) for more options.
+
+### Send emails using Outlook.com
+
+After setting the required fields above:
+
+1. Setup an account on Microsoft 365 or outlook.com
+2. Set the following configuration options:
+    ```
+    CODER_NOTIFICATIONS_EMAIL_SMARTHOST=smtp-mail.outlook.com:587
+    CODER_NOTIFICATIONS_EMAIL_TLS_STARTTLS=true
+    CODER_NOTIFICATIONS_EMAIL_AUTH_USERNAME=<user>@<domain>
+    CODER_NOTIFICATIONS_EMAIL_AUTH_PASSWORD="<account password>"
+    ```
+
+See [this help article from Microsoft](https://support.microsoft.com/en-us/office/pop-imap-and-smtp-settings-for-outlook-com-d088b986-291d-42b8-9564-9c414e2aa040)
+for more options.
+
+## Webhook
+
+The webhook delivery method sends an HTTP POST request to the defined endpoint. The purpose of webhook notifications is
+to enable integrations with other systems.
```

Comment:
I'd call out Slack explicitly so `ctrl+F` takes them straight to the webhook payload.
```
...with other systems such as Slack. 
```

Reply:
Our webhook format is not the same as Slack's; folks will have to create their own custom integration for that.
I've created https://github.com/coder/team-coconut/issues/47 for this; it'll take a bit of work.